_migrate-or-upgrade/rolling-upgrade/index.md

```diff
@@ -18,8 +18,7 @@ Rolling upgrades, sometimes referred to as "node replacement upgrades," can be p
 This document serves as a high-level, platform-agnostic overview of the rolling upgrade procedure. For specific examples of commands, scripts, and configuration files, refer to the [Rolling upgrade lab]({{site.url}}{{site.baseurl}}/migrate-or-upgrade/rolling-upgrade/rolling-upgrade-lab/).
 
 ## Preparing to upgrade
-
-Review [Upgrading OpenSearch]({{site.url}}{{site.baseurl}}/migrate-or-upgrade/rolling-upgrade/) for recommendations about backing up your configuration files and creating a snapshot of the cluster state and indexes before you make any changes to your OpenSearch cluster.
+Before making any changes to your OpenSearch cluster is it highly recommended to back up your configuration files and create a [snapshot]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore/) of the cluster state and indexes.
```

Comment:
Note that OpenSearch only allows rolling upgrades from major adjacent versions. For example, a rolling upgrade can be performed from OpenSearch 1.x to 2.x, but not from OpenSearch 1.x to 3.x.